python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -279,7 +339,9 @@ def flatBiasIngest(repo, calib_repo, calibdatafiles):
     BASH EQUIVALENT:
     $ ingestCalibs.py repo --calib calib_repo --mode=link --validity 999 calibdatafiles
     '''
-    print('Ingesting flats and biases...')
+    lsst.log.configure()
```

Comment:
Shouldn't this call live in your `main` function? Why should `flatBiasIngest` be responsible for setting up logging?

Reply:
I presently have `lsst.log.configure()` in each of the main `ap_pipe` functions (seven total instances, in fact). I assumed that each function needed to have its own logging setup, but that might not be right.

Reply:
I'm pretty sure you're only supposed to do it once per program; the LSST documentation isn't particularly clear, but most loggers work that way.

Reply:
OK nevermind :)